check/check.go

```diff
@@ -212,6 +219,10 @@ func (c *Check) execute() (finalOutput *testOutput, err error) {
 		if !result.flagFound {
 			t.isConfigSetting = true
 			result = *(t.execute(c.AuditConfigOutput))
+			if !result.flagFound && t.Env != "" {
+				t.isConfigSetting = false
```

Comment:
Currently if `isConfigSetting` is false it means we get it from running `c.auditOutput`, and if it's true it means we get it from running `c.auditConfigOutput`. I think we need to change this from a boolean to an enum so that it can take three options: `audit`, `auditConfig` or `auditEnv`. Maybe we change the first one to be `auditCommand` for even more clarity. 

Then this enum is what you can use in `t.findValue()` (see other comment)

Reply:
✔️